Doctrine/Phpcr/MediaManager.php

```diff
@@ -133,8 +134,12 @@ public function setDefaults(MediaInterface $media, $parentPath = null)
      */
     public function mapPathToId($path, $rootPath = null)
     {
-        // The path is being the id
-        $id = PathHelper::absolutizePath($path, '/');
+        try {
+            // The path is being the id
+            $id = PathHelper::absolutizePath($path, '/');
+        } catch (RepositoryException $e) {
+            throw new \OutOfBoundsException($e->getMessage());
```

Comment:
oh, and when wrapping exceptions, we have to pass the original exception in the arguments, or we hide important information when debugging.